test/gtest/common/test_helpers.cc

```diff
@@ -98,6 +98,14 @@ bool is_inet_addr(const struct sockaddr* ifa_addr) {
            (ifa_addr->sa_family == AF_INET6);
 }
 
+const struct sockaddr *
+sockaddr_set_port(const struct sockaddr_storage& ss, uint16_t port) {
+    struct sockaddr_in *sockaddr_in_p = (struct sockaddr_in *)&ss;
```

Comment:
do we need to have support for `AF_INET6` af?
```
switch (ss.ss_family) {
case AF_INET:
    ((struct sockaddr_in *)&ss)->sin_port = port;
    break;
case AF_INET6:
    ((struct sockaddr_in *)&ss)->sin6_port = port;
    break;
default:
    return NULL;
}
```

I see that `is_inet_addr` has recently been updated by you to support `AF_INET6`.

Reply:
> do we need to have support for AF_INET6 af?

yes

> I see that is_inet_addr has recently been updated by you to support AF_INET6

good catch, thanks! will fix